downstream/modules/platform/ref-controller-intro-job-template.adoc

```diff
@@ -16,7 +16,7 @@ This ensures IT teams can reliably scale automation while maintaining high effic
 
 The default view is collapsed (Compact), showing the template name, template type, and the timestamp of the last job that ran using that template. You can click the image:arrow.png[Arrow,15,15] icon next to each entry to expand and view more information. This list is sorted alphabetically by name, but you can sort by other criteria, or search by various fields and attributes of a template.
 
-From this screen you can launch image:rightrocket.png[Launch icon,15,15] , edit image:leftpencil[Edit icon,15,15], and duplicate image:copy.png[Duplicate icon] a job template
+From this screen you can launch image:rightrocket.png[Launch icon,15,15] , edit image:leftpencil.png[Edit icon,15,15], and duplicate image:copy.png[Duplicate icon] a job template
```

Comment:
A period is needed at the end of this sentence. Also, I didn't run a preview yet, but the rich diff shows "EditIconEditIcon" and I'm not sure if it publishes that way? I don't know if there is a coding error there.